<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Вводимые данные</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="3" column="0">
              <widget class="QLabel" name="wall_hight_lbl">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Hнс, мм:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lenght_wall_lbl">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Lнс, мм:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="width_lbl">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Wнс, мм:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="full_hight_lbl">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Hнт, мм:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="hight_full_doubleSpinBox">
               <property name="toolTip">
                <string>Высота наружнего торца(конек)</string>
               </property>
               <property name="whatsThis">
                <string>Высота наружнего торца(конек)</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="hight_wall_doubleSpinBox">
               <property name="toolTip">
                <string>Высота стенки</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="lenght_walldoubleSpinBox">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="width_doubleSpinBox">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="back_btn">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Назад</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="next_btn">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Далее</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Внешний тент:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="count_btn">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Рассчет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Итоговые данные:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QFormLayout" name="formLayout_2">
             <property name="rowWrapPolicy">
              <enum>QFormLayout::WrapLongRows</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>S наружнего тента, м2:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>S внутреннего тента, м2:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>S общая, м2:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Графический вид</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Вводимые данные</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="3" column="0">
              <widget class="QLabel" name="wall_hight_lbl_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Hнс, мм:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lenght_wall_lbl_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Lнс, мм:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="width_lbl_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Wнс, мм:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="full_hight_lbl_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Hнт, мм:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="hight_full_doubleSpinBox_2">
               <property name="toolTip">
                <string>Высота наружнего торца(конек)</string>
               </property>
               <property name="whatsThis">
                <string>Высота наружнего торца(конек)</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="hight_wall_doubleSpinBox_2">
               <property name="toolTip">
                <string>Высота стенки</string>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="lenght_walldoubleSpinBox_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="width_doubleSpinBox_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="maximum">
                <double>1000000000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="back_btn_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Назад</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="next_btn_2">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Далее</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_9">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>Внешний тент:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="count_btn_2">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Рассчет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Итоговые данные:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QFormLayout" name="formLayout_4">
             <property name="rowWrapPolicy">
              <enum>QFormLayout::WrapLongRows</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>S наружнего тента, м2:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>S внутреннего тента, м2:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_13">
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
               </property>
               <property name="text">
                <string>S общая, м2:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
            </property>
            <property name="text">
             <string>Графический вид</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView_2"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
